docs/examples/advanced-cue/podenv.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: core.oam.dev/v1alpha2
+kind: TraitDefinition
+metadata:
+  annotations:
+    definition.oam.dev/description: "add env into your pods"
+  name: podenv
```

Comment:
`env` is fine, it's inside the application. For deployment environment, we should use `environment`.